PrimerasPracticasJS/NumeroDeVocales.js

```diff
@@ -0,0 +1,22 @@
+/*SACAR EL NUMERO DE VOCALES*/
+
+let text = 'Hola Mundo soy Jona Crack'
+
+text = text.toUpperCase()
+
+let textArray = text.split('')
+
+function countVowels(text) {
+    let vowels = ['A', 'E', 'I', 'O', 'U']
+    let numVowels = 0
+
+    for (let i = 0; i < textArray.length; i++) {
+        if (vowels.includes(textArray[i])) {
+            numVowels++
+        }//end if
+    }//end for
+
+    return numVowels
+}
```

Comment:
el uppercase y split forman parte del proceso de countVowels, creo seria buena idea que vayan dentro de la misma funcion, tambien se pueden concatenar metodos por ejemplo el toUpperCase con el split para que quede todo en 1 sola linea.

```suggestion
function countVowels(text) {
    let vowels = ['A', 'E', 'I', 'O', 'U']
    let numVowels = 0 
    let textArray = text.toUpperCase().split('')

    for (let i = 0; i < textArray.length; i++) {
        if (vowels.includes(textArray[i])) {
            numVowels++
        }//end if
    }//end for

    return numVowels
}
```